src/main/resources/org/sonatype/nexus/ci/nxrm/Messages.properties

```diff
@@ -21,3 +21,11 @@ NexusPublisherWorkflowStep.FunctionName=nexusPublisher
 NexusPublisherWorkflowStep.DisplayName=Nexus Repository Manager Publisher
 
 NexusPublisherBuildStep.DisplayName=Nexus Repository Manager Publisher
+
+CreateTag.DisplayName=Create Tag (Nexus Repository Manager 3.x)
+CreateTag.Validation.TagNameEmpty=Tag name cannot be blank
```

Comment:
Question: Do we want to be consistent with the format of the display name by leading with the prefix `Nexus Repository Manager` as the existing steps or opt for the approach of appending `(Nexus Repository Manager 3.x)` ?

Reply:
The NexusPublisher (unless changed) will be both NXRM2 and NXRM3. I think it's helpful to provide the additional description of the NXRM3 text for clarity of the target for the build step. That said it's only helpful when used from the UI, and if we can find a way to prevent the step from showing up for NXRM2 targets that'd be great as an alternative.